.codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1%
```

Comment:
@jeff-regier , what does this threshold set?

Reply:
It sets how much the overall pytest coverage can decrease before a red x (rather than a green check) appears next to a pull request. This way the red x really means something when it appears.